websites/Z/ZenMarket/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "82662823523516416",
+		"name": "maroxy"
+	},
+	"service": "ZenMarket",
+	"description": {
+		"en": "ZenMarket is a Japanese Proxy Service"
+	},
+	"url": "zenmarket.jp",
+	"version": "1.0.0",
```

Comment:
There is a new field called `matches`, which is required for new Presences.
These are an array of strings which need to follow https://developer.chrome.com/docs/extensions/develop/concepts/match-patterns.